Makefile

```diff
@@ -14,8 +14,8 @@
 
 EXTENSION_PREFIX            := gardener-extension
 NAME                        := networking-cilium
-REGISTRY                    := eu.gcr.io/gardener-project
```

Comment:
Please keep the gardener registry and image prefix. I will try out the changes later. Thx for the PR.

Reply:
Argh, that leaked in, thank you for catching that. Fixed and rebased.